src/app/pages/server-compare/server-compare.component.ts

```diff
@@ -216,12 +225,18 @@ export class ServerCompareComponent implements OnInit, AfterViewInit {
     private route: ActivatedRoute) { }
 
   ngOnInit() {
+    const id = this.route.snapshot.queryParamMap.get('id');
```

Comment:
can we please use url param instead of a query parameter here?

`/compare/foobar` instead of `/compare?id=foobar`